src/client/LiveConf.js

```diff
@@ -74,6 +69,7 @@ const LiveconfTemplate = {
         part_ttl: 3600,
         playout_type: "live",
         source_timescale: null,
+        reconnect_timeout: 600,
```

Comment:
It looks like your `develop` branch was a little outdated. The reconnect_timeout stuff is in the latest